### DISCUSSION TITLE
Welcome to pymobiledevice3 Discussions!

### DISCUSSION THREAD

**Discussion by doronz88** [doronz88/pymobiledevice3#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rvillarreal-r7** [comment]
Hello, Thank you for the work done on this project! Outside of the Un-implemented services what else is needed to support the project? I'd like to work on adding features, but i'm not sure where to begin.

**Comment by doronz88** [comment]
Thank you @rvillarreal-r7 . We are always happy to receive PRs. If you are willing, I think XCUITest is probably the most wanted feature from most people so implementing it could be really nice
